docs/readmes/orc8r/dev_minikube.md

```diff
@@ -44,6 +44,7 @@ There are 2 ways you can publish your own images: to a private registry, or to a
 2. Publish to localhost registry (specifically, we'll use the [local Docker registry via Minikube](https://minikube.sigs.k8s.io/docs/handbook/registry/#docker-on-macos))
     - `minikube addons enable registry`
     - `docker run --rm -it --network=host alpine ash -c "apk add socat && socat TCP-LISTEN:5000,reuseaddr,fork TCP:$(minikube ip):5000"`
+      - This should hang, open new tab for other commands
```

Comment:
```suggestion
    - (This should hang, open new tab for other commands)
```

Reply:
Oh I did the tabbing on purpose so that it's obvious that the `This should hang, open new tab for other commands` refers to the `docker run --rm -it --network=host alpine ash -c "apk add socat && socat TCP-LISTEN:5000,reuseaddr,fork TCP:$(minikube ip):5000"` command
![image](https://user-images.githubusercontent.com/29292203/141335104-de3fef23-9316-484e-a444-c2942721f6a4.png)

Reply:
Ah gotcha that's great. Just indent with 4 spaces instead of 2, then 👍